dev/CommandBarFlyout/CommandBarFlyoutCommandBar.cpp

```diff
@@ -562,6 +562,7 @@ void CommandBarFlyoutCommandBar::UpdateVisualState(
     }
     else
     {
+        winrt::VisualStateManager::GoToState(*this, L"Default", useTransitions);
```

Comment:
What is the purpose of this line?

Reply:
I added bunch of visual state setter in that group to change corner radius.

Bottom corner radius of the primary menu are set to 0 when secondary menu opens up, and the values need to be set back to default when secondary menu closes.

Reply:
That VisualStateGroup was used to change the border thickness of the secondary menu(popup), we didn't need to set the value back because it's not visible when popup closes. We need to set it back to default now since that group also includes primary menu corner changes, which is still visible when popup closes.